ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/event-handlers.j2

```diff
@@ -75,9 +75,26 @@ event-handler {{ handler.name }}
       regex {{ handler.regex }}
 {%             endif %}
 {%         endif %}
-{%         if handler.action is arista.avd.defined and handler.action_type is arista.avd.defined %}
+{%         if handler.action_type is arista.avd.defined("bash") and handler.action is arista.avd.defined %}
```

Comment:
Let's not add the check for bash, since users might have "misused" this. I suggest just keeping the old logic untouched.

Reply:
Fixed